bin/hof-transpiler

```diff
@@ -84,22 +91,24 @@ _.each(sources, function (src) {
     _.each(writeFileStream, function (writerStream) {
       var sharedMap = {};
 
-      walk.walk(shared).on('file', function (root, fileStats, next) {
-        var lang = getLanguage(root);
-        var fileName = fileStats.name.replace('.json', '');
-        var streamLang = getLanguage(path.parse(writerStream.path).dir);
-        var parsedStream = null;
-        var stringStream = '';
+      shared.forEach(function (sharedSource) {
+        walk.walk(sharedSource).on('file', function (root, fileStats, next) {
+          var lang = getLanguage(root);
+          var fileName = fileStats.name.replace('.json', '');
+          var streamLang = getLanguage(path.parse(writerStream.path).dir);
+          var parsedStream = null;
+          var stringStream = '';
```

Comment:
No that would be a separate issue, not in the scope of this. Having a half es5 half es6 file is never a good thing